test/test_dht.cpp

```diff
@@ -1069,7 +1069,7 @@ TORRENT_TEST(dht)
 		printf("num_active_buckets: %d\n", tbl.num_active_buckets());
 		TEST_EQUAL(tbl.num_active_buckets(), 6);
    
-#if defined TORRENT_DHT_VERBOSE_LOGGING || defined TORRENT_DEBUG
```

Comment:
I actually deliberately left these disabled until they do something more reasonable. Back when verbose logging was off by default, and just a debug feature, it was fine for these to print to cerr. but printing to cerr seems intrusive and odd by default